### PR TITLE
[plugin.video.viaplay] 1.0.5

### PR DESCRIPTION
**Pull request by emilsvennesson**

### CODE REVIEW

File: plugin.video.viaplay/resources/lib/vialib.py [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+            f.write(subtitle)
+            f.close()
+            subtitle_paths.append(path)
+
+        return subtitle_paths
+
+    def get_deviceid(self):
+        """"Read/write deviceId (generated UUID4) from/to file and return it."""
+        try:
+            deviceid = open(self.deviceid_file, 'r').read()
```
Comment (phil65) [inline plugin.video.viaplay/resources/lib/vialib.py]: please use "with" statement for opening files.

File: plugin.video.viaplay/resources/lib/vialib.py [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+
+            if '_sv' in suburl:
+                path = os.path.join(self.tempdir, 'swe.smi')
+            elif '_no' in suburl:
+                path = os.path.join(self.tempdir, 'nor.smi')
+            elif '_da' in suburl:
+                path = os.path.join(self.tempdir, 'dan.smi')
+            elif '_fi' in suburl:
+                path = os.path.join(self.tempdir, 'fin.smi')
+            f = open(path, 'w')
```
Comment (phil65) [inline plugin.video.viaplay/resources/lib/vialib.py]: please use "with" statement for opening files.

**Comment by phil65** [comment]
two comments, rest looks fine to me.

**Comment by emilsvennesson** [comment]
@phil65 Fixed. :)

Changes since last review: https://github.com/emilsvennesson/kodi-viaplay/compare/v1.0.3...v1.0.4

**Comment by phil65** [comment]
thx, looks fine.

**Comment by emilsvennesson** [comment]
@phil65 Is there a reason why this hasn't been merged? :-)

**Pull request merged**